src/components/EntityInfo.vue

```diff
@@ -25,8 +25,8 @@
             </div>
 
             <!-- Entity Type -->
-            <div v-if="entityDescription" id="entity-description">{{ entityDescription }}</div>
-            <div v-if="nrDescription" id="nr-subtitle">{{ nrDescription }}</div>
+            <div v-if="businessId" id="entity-description">{{ entityDescription }}</div>
+            <div v-if="tempRegNumber" id="nr-subtitle">{{ nrDescription }}</div>
```

Comment:
This is also a bugfix (undocumented). Previously, both lines would display.

Reply:
Nice to catch these before making it shared :) 
I imagine the Filings UI will be the baseline from which we build our Shared version.